samples/secure-fraud-detection/src/main/java/io/quarkiverse/langchain4j/sample/CustomerRepository.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkiverse.langchain4j.sample;
+
+import io.quarkus.hibernate.orm.panache.PanacheRepository;
+import jakarta.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class CustomerRepository implements PanacheRepository<Customer> {
+    /*
+     * Transaction limit for the customer.
+     */
+	public int getTransactionLimit(String customerName, String customerEmail) {
+		return find("name = ?1 and email = ?2", 
```

Comment:
Depending on both name AND email can only lead to additional problems without adding any value, email should be enough, no?

Reply:
@jmartisk This is meant to show how several pieces from the current Security Identity can be passed around and used for something, like formulating a query. It is not meant as some concrete prod-quality recommendation (and I'm not convinced that there is 0 value in combining both, but I don't really mind, name can be dropped if you prefer).

What having both values does show though is that having a custom memory id transformed to String at the source makes it more difficult to restore though values, which is why I meant to ask @geoand, once it is approved otherwise, if some workaround can be found, but it can be discussed later for sure

Reply:
Let me resolve it for now, happy to revisit a bit later once other issues are sorted out, with PR still being open :-)